Gruntfile.js

```diff
@@ -22,7 +22,6 @@ module.exports = function (grunt) {
           '<%= srcPath %>/XmlUtil.js',
           '<%= srcPath %>/Request.js',
           '<%= srcPath %>/Filter.js',
-          '<%= srcPath %>/Filter.GmlObjectID.js',
```

Comment:
Code simplification and understanding. Each feature in separate file. In future there are will be some another filters.